src/index.js

```diff
@@ -312,47 +299,62 @@ async function cleaningUnwantedElements() {
     })
     .indexFields(['cozyMetadata.createdByApp'])
   const dirsResults = await cozyClient.new.queryAll(dirsQuery)
-  for (const result of dirsResults) {
-    if (months.some(month => result.attributes.name.startsWith(month))) {
-      dirsToDelete.push(result.id)
-    }
-  }
-  for (const directoryId of dirsToDelete) {
-    const filesQuery = Q('io.cozy.files')
-      .where({
-        dir_id: directoryId
-      })
-      .partialIndex({
-        trashed: false,
-        type: 'file'
-      })
-    const filesResults = await cozyClient.new.queryAll(filesQuery)
-    for (const file of filesResults) {
-      filesInDirsToDelete.push(file)
+  for (const line of linesAndNames) {
+    const brokenName = `${line[0]} ()`
+    const correctName = `${line[0]} (${line[1]})`
+    for (const dir of dirsResults) {
+      if (
+        dir.name === brokenName &&
+        // Checking existence of destination directory
+        dirsResults.some(currentDir => currentDir.name === correctName)
+      ) {
+        log('debug', 'Need to clean one directory')
+        const idDestination = dirsResults.find(
```

Comment:
nit: You could do this `dirsResults.find` once and use it in place of the `dirsResults.some` line 309, no ? To avoid to have the same loop twice.

Reply:
Done